frontend/src/Editor/Editor.jsx

```diff
@@ -141,13 +142,23 @@ class Editor extends React.Component {
     this.fetchApps(0);
     this.fetchApp();
     this.initComponentVersioning();
+    this.initRealtimeSave();
     this.initEventListeners();
     this.setState({
       currentSidebarTab: 2,
       selectedComponent: null,
     });
   }
 
+  initRealtimeSave = () => {
+    this.props.ymap.observe(() => {
```

Comment:
Think it'll be helpful to add a comment here to signify why ymap check is being done

Reply:
Added a comment ^^